ui/src/gallery/components/MetaBoxByType/MultiKSampler.tsx

```diff
@@ -0,0 +1,66 @@
+import {clipboard, ifString, MetaBoxTypeCom, MetaData} from '../../utils.ts'
+import {Box, Button, Flex, IconButton, Input, Textarea, VStack} from '@chakra-ui/react'
+import {IconCopy} from '@tabler/icons-react'
+
+export const isMultiKSampler = (metaData: MetaData) => {
+  return Object.keys(metaData?.prompt)?.filter?.(v => metaData?.prompt?.[v]?.['class_type'] === 'KSampler')?.length > 1
+}
+export const calcMultiKSampler: (metaData: MetaData) => {[key: string]: string} = (metaData: MetaData) => {
+  const {prompt} = metaData
+  const promptKeys = Object.keys(prompt)
+  const kSamplerKeys = promptKeys?.filter(v => prompt?.[v]?.['class_type'] === 'KSampler')
+  return kSamplerKeys.reduce((previousValue, currentValue, currentIndex) => {
+    const kSampler = prompt?.[currentValue]
+    const ckptName = ifString(prompt?.[kSampler?.inputs?.['model']?.[0]]?.inputs?.['ckpt_name']) ?? ifString(prompt?.[Object.keys(prompt)?.find((v: string) => prompt[v]?.inputs?.ckpt_name) ?? '']?.inputs?.ckpt_name) ?? ''
+    const positive = ifString(prompt?.[kSampler?.inputs?.['positive']?.[0]]?.['inputs']?.['text'])
```

Comment:
i think the multi k sampler and single k sampler can share lots of common code, can try to reuse code as much as possible to reduce bundle side? we need to be careful to keep out bundle size small because some users use our app from retend cloud GPU with shared bandwidth and its not fast